discovery-provider/src/queries/get_feed.py

```diff
@@ -1,5 +1,7 @@
 import datetime
```

Comment:
how do you feel about a `get_feed` with a `get_feed_sql` and `get_feed_es` separation of files?